contrib/upgrade_testnet/start.sh

```diff
@@ -15,7 +15,17 @@ rm -r -f $BUILDDIR
 
 # Create the 4 nodes folders with the correct denom
 echo "===> Creating $NODES nodes localnet"
-make setup-localnet COIN_DENOM="udaric" NODES=$NODES > /dev/null > /dev/null
```

Comment:
@RiccardoM I tried editing this in order to make it use the docker compiled binary for the nodes, instead of using setup-localnet that use make build-linux (without linking the correct static library needed).
I'm now having some errors inside the workflow, in particular these:
https://github.com/desmos-labs/desmos/runs/5035433674?check_suite_focus=true#step:5:330

https://github.com/desmos-labs/desmos/runs/5035433674?check_suite_focus=true#step:5:385

https://github.com/desmos-labs/desmos/runs/5035433674?check_suite_focus=true#step:5:388

https://github.com/desmos-labs/desmos/runs/5035433674?check_suite_focus=true#step:5:391

https://github.com/desmos-labs/desmos/runs/5035433674?check_suite_focus=true#step:5:394

I'm not sure why these are happening 🤔 I've also tried to add sudo in front of the cmd but it din't work.
Can you take a look on this?

Reply:
First of all, I would suggest you removing the usage of `--gentx-coin-denom="udaric"` since it's not necessary. The `setup-genesis.py` script that is run will take care of replacing all the `stake` coin denoms with the one you provide with `COIN_DENOM` so that's sufficient without editing the previous `testnet` command. 

That being said, this error is due to file permissions. You need to run the Docker command passing to it the same user that owns all those files. You can do so by adding the `--user $UID:$GID` to the `docker run` command. 

Finally, please don't use `sudo` on Docker commands, and try using `make` commands (eg. `make setup-localnet`) instead of duplicating the code by copy/pasting the commands

Reply:
@RiccardoM I would like to use the `make` one, but it looks like it's not possible since we need to include the static library during the build process.